provisioning/azure/scripts/deploy.sh

```diff
@@ -17,16 +17,26 @@ kubectl delete all -n default -l app=templates-api --ignore-not-found
 kubectl delete configmap -n default templates-api --ignore-not-found
 
 kubectl apply -f ./provisioning/kubernetes/deploy/namespace.yaml
+kubectl apply -f ./provisioning/kubernetes/deploy/etcd.yaml
 kubectl apply -f ./provisioning/kubernetes/deploy/config-map.yaml
 kubectl apply -f ./provisioning/kubernetes/deploy/templates-api.yaml
 kubectl apply -f ./provisioning/kubernetes/deploy/azure/service.yaml
 kubectl rollout status deployment/templates-api --timeout=900s
 
-TEMPLATES_API_IP=$(kubectl get services \
-  --selector "app=templates-api" \
-  --namespace templates-api \
-  --no-headers \
-  --output jsonpath="{.items[0].status.loadBalancer.ingress[0].ip}")
+TEMPLATES_API_IP=""
+TIME_WAITED=0
+# every 10 seconds but in total max 15 minutes try to fetch TEMPLATES_API_IP
+#shellcheck disable=2203
+while [[ -z "$TEMPLATES_API_IP" && $TIME_WAITED -lt 15*60 ]]; do
+    echo "Waiting for Templates API ingress IP..."
+    sleep 10;
+    TIME_WAITED=$((TIME_WAITED + 10))
+    TEMPLATES_API_IP=$(kubectl get services \
+      --selector "app=templates-api" \
+      --namespace templates-api \
+      --no-headers \
+      --output jsonpath="{.items[0].status.loadBalancer.ingress[0].ip}")
+done
```

Comment:
Neviem teda, ci bol problem aj v tomto alebo nie, ale zostat to tu moze:
https://github.com/keboola/keboola-as-code/pull/770#issuecomment-1225872234

Reply:
to se hodí, dal jsem to už do #778